include/mbedtls/aes.h

```diff
@@ -69,26 +69,49 @@ extern "C" {
 // Regular implementation
 //
 
+/** VIA padlock needs 16 extra Bytes for alignment. */
+#if (defined(MBEDTLS_PADLOCK_C) && defined(MBEDTLS_HAVE_X86)) ||        \
+    (defined(MBEDTLS_AESNI_C) && MBEDTLS_AESNI_HAVE_CODE == 2)
+#define MBEDTLS_PADLOCK_EXTRA 4
+#else
+#define MBEDTLS_PADLOCK_EXTRA 0
+#endif
+
+/** Each round key occupies 16 Bytes. */
+#define MBEDTLS_AES_RK_UINT32S 4
+
+/** Define number of maximum rounds in AES key expansion.
+ *   See https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.197.pdf
+ *      - Section 5, Nr = Nk + 6
+ *      - Section 5.2, the length of round keys is Nb*(Nr+1)
+ */
+#if defined(MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH)
+#define MBEDTLS_AES_MAX_ROUNDS 10
+#else
+#define MBEDTLS_AES_MAX_ROUNDS 14
+#endif
+
+/** Allocate 8 extra Bytes for AES key expansion.
+ *   - 16 extra Bytes to store initial round key.
+ *   - 16 extra Bytes to simplify key expansion in the 256-bit case.
+ */
+#define MBEDTLS_AES_RK_EXTRA (8 + MBEDTLS_PADLOCK_EXTRA)
```

Comment:
If this 8 is the "8 extra bytes" then it's wrong, since this is used as a number of `uint32_t`s